fetch/fetchRequest.js

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+require('es6-promise').polyfill();
+require('isomorphic-fetch');
+
+function fetchRequest(method, url, args) {
+  var f = fetch(url, {
+    method: method,
+    headers: args.headers,
+    body: JSON.stringify(args.data),
+    credentials: 'include'
```

Comment:
Good resource for understanding how CORS and `credentials: include` work together: https://fetch.spec.whatwg.org/#cors-protocol-and-credentials

It is named kinda confusingly, as in this case it actually refers to whether _cookies_ will be sent by fetch. Since this use case is backchannel, not frontchannel, I think we could actually ignore cookies. Would like @robertdamphousse-okta's thoughts.

Reply:
Sorry @nbarbettini, maybe this is confusing because I haven't removed reqwest yet, but the goal is to then remove reqwest and use this for both the backchannel and the frontchannel, since isomorphic-fetch is indeed isomorphic (and much more modern, and native, than reqwest). This is why I added `credentials: 'include'`, to be consistent with the one we are currently using in reqwest: https://github.com/okta/okta-auth-js/blob/master/reqwest/reqwestRequest.js#L21

Reply:
I think we should handle the credentials-or-not change as a separate PR, this one should focus on the cross-platform support.